labview/labview_types.h

```diff
@@ -0,0 +1,39 @@
+#include "extcode.h"
+
+#ifndef LABVIEW_TYPES_H
+#define LABVIEW_TYPES_H
+
+/**
+ * This function copies the contents of a C string into a LabVIEW LStrHandle,
+ * resizing the handle if necessary.
+ */
+MgErr copyStringToLStrHandle(char *cpString, LStrHandle LVString);
+
+/**
+ * This function copies the contents of a binary buffer into a LabVIEW
+ * LStrHandle, resizing the handle if necessary. The function is designed for
+ * handling binary data.
+ */
+MgErr copyBufferToLStrHandle(const void *buffer, int len, LStrHandle LVString);
+
+/**
+ * This function searches for a specified character within a binary buffer and
+ * returns a pointer to the first occurrence of the character, starting from the
+ * given offset.
+ */
+char* findInBuffer(const uint8_t* headerBuffer, uint32_t headerBufferLen, uint32_t offset, char toFind);
+
+/**
+ * Split a string into an array of tokens based on a delimiter.
+ * Memory is allocated for the token array, and the caller is responsible for
+ * freeing the memory when it's no longer needed.
+ */
```

Comment:
you said split string based on delimiter, but you don't pass a delimiter as param, is that normal?

Reply:
delimiter is hardcoded in this function

Reply:
in the C file it does not seems to be hardcoded

Reply:
ah true, It was hardcoded then I changed that in the c file. I will close this one - because we had another conversation related to the incorrect definition of this function in the header - an that exactly is the bug to fix.